docs/src/main/sphinx/develop/connectors.rst

```diff
@@ -303,8 +303,14 @@ query.
 Predicate pushdown
 ^^^^^^^^^^^^^^^^^^
 
-When executing a query with a ``WHERE`` clause, the query plan may
-contain a ``ScanFilterProject`` operation.
+When executing a query with a ``WHERE`` clause, the query plan will likely
+contain a ``ScanFilterProject`` operation with a predicate constraint.
+
+A predicate constraint is a description of the constraint imposed on the
+results of the query as expressed in the ``WHERE`` clause. For example
+``WHERE x > 5 AND y = 3`` would translate into a ``Constraint`` where the
+``summary`` field would say the ``x`` column's domain has to be greater than
```

Comment:
- No use of "would"
- Do not use "say" - google style guide requires plain, non-idiomatic language, and inanimate objects don't actually say things. 
 
- You can use "means" instead of "would say".

Use "must be" instead of "has to be"